javascript/javascript2/week1/index.html

```diff
@@ -8,25 +8,25 @@
     <title>Document</title>
 </head>
 <body>
-    <!-- <input type="radio" id="button" name="event-types" value="button">
-    <label for="button"> Button</label><br>
-
-    <input type="radio" id="hover" name="event-types" value="button">
-    <label for="hover"> Hover</label><br>
+    <div id="root">
+        <h2> What would you like to see your spirit name with? </h2>        
+        <select id="animal-selection">
+            <option value="click">Button</option>
+            <option value="mouseover">Hover</option>
+            <option value="keyup">Text</option>
+        </select>
 
-    <input type="radio" id="textBox" name="event-types" value="button">
-    <label for="textBox"> Text Box</label><br> -->
+    </div>
 
-    <div id="root">
+    <div class="hoverSelected">
         <h2>Please Enter Your Name here: </h2>
-        <input type="text" id="nameArea" placeholder="Find my spirit animal" required>       <!-- create input element -->            
-        <button type="submit" onclick="getUserName(event)"> Summon </button>                
+        <input type="text" id="nameArea" onmouseover="generateAnimalName(event)" onkeyup="generateAnimalName(event)" placeholder="Find my spirit animal" required>
```

Comment:
Here =)

Reply:
This is not correct I don't know if you try to run it yourself? try to create 3 different `EventListener` in your `index.js`, and you should use `DOM` instead of putting function in HTML file. 

I suggest you next time when you have many exercises, try not to put in one file like you did in `index.js`, create file for each exercises.

Reply:
Yes of course I tested and it works, without test how can I know :)
Will change to with EventListner.
Thanks for your advice.